crates/core/plugin_sm/src/plugin.rs

```diff
@@ -421,19 +429,19 @@ impl Plugin for ExternalPluginCommand {
             let mut rdr = ReaderBuilder::new()
                 .has_headers(false)
                 .delimiter(b'\t')
+                .flexible(true)
                 .from_reader(output.stdout.as_slice());
 
             for module in rdr.deserialize() {
```

Comment:
It would be good to extract this whole deserialization in a function - this will help unit testing. The current tests are okay but made more complex than necessary because requiring a fake plugin.